addons/web/static/src/legacy/js/views/calendar/calendar_controller.js

```diff
@@ -390,6 +390,13 @@ var CalendarController = AbstractController.extend({
                     self.reload();
                 },
             });
+            this.previousOpen.on('closed', this, () => {
```

Comment:
This is not good, you're creating a dependency between renderer and controller.
In the renderer when you call `trigger_up("openCreate", something)` you should pass a callback to something that will do the unselect.

don't forget to add a test